src/Resources/app/storefront/src/scss/category-extend.scss

```diff
@@ -0,0 +1,75 @@
+.category-filter-container {
+  min-width: 290px;
+  label {
+    display: inline-block;
+    width: auto;
+    padding: 0px;
+  }
+}
+
+.show-category-list-item {
+  max-height: 500.313rem;
+  min-height: 1.3125rem;
+  height: auto;
+  opacity: 1;
+  transition: all 0.5s;
+  width: 109%;
+}
+
+.up-arrow {
+  transform: rotate(180deg);
+  height: 30px;
+  width: 35px;
+  display: inline-block;
+  left: -5px;
+  top: 0px;
```

Comment:
Don't use `left` or `top` values, unless the element is positioned `absolute`. Also I don't think that a `top: 0px` is even needed at all.

Reply:
The position changed to "absolute", "left" property has been removed,` top: 0px` is there because of the transition, that is coming from `top: -5px` to `top: 0px` this is why it is necessary to mention.